src/components/MultiTagSelection.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+import Downshift from 'downshift'
+
+class MultiDownshift extends React.Component {
+  state = {selectedItems: []}
+
+  stateReducer = (state, changes) => {
+    switch (changes.type) {
+      case Downshift.stateChangeTypes.clickItem:
+        return {
+          ...changes,
+          isOpen: true,
+        }
+      default:
+        return changes
+    }
+  }
+
+  handleSelection = (selectedItem, downshift) => {
+    const callOnChange = () => {
+      if (this.props.onSelect) {
+        this.props.onSelect(
+          this.state.selectedItems,
+          this.getStateAndHelpers(downshift),
+        )
+      }
+      if (this.props.onChange) {
+        this.props.onChange(
+          this.state.selectedItems,
+          this.getStateAndHelpers(downshift),
+        )
+      }
+    }
```

Comment:
Prefer adding an empty line here to make it clearer.

Reply:
modified